packages/react/src/components/Search/next/Search.js

```diff
@@ -95,7 +100,17 @@ function Search({
 
   return (
     <div role="search" aria-labelledby={searchId} className={searchClasses}>
-      <div className={`${prefix}--search-magnifier`} ref={magnifier}>
+      {/* the magnifier is used in ExpandableSearch as a click target to expand, 
+      however, it does not need a keyboard event bc the input element gets focus on keyboard nav and expands that way*/}
+      {/* eslint-disable-next-line jsx-a11y/click-events-have-key-events */}
+      <div
```

Comment:
Is it possible for you to change this element to a native `button`? I think `role="button"` implies a lot of semantics that we don't really provide here

Reply:
@joshblack it doesn't function as a button in our default `Search`, only in the `ExpandableSearch`, which is why I didn't change it. It's kinda tricky. We also don't use it as a button in keyboard expandable search, it gets completely overlooked and the input gets focus.

Reply:
In terms of screen reader a11y, I tested it with VoiceOver, and it's working as expected.